functions.php

```diff
@@ -38,21 +36,9 @@ function register_my_menu() {
 add_action( 'init', 'register_my_menu' );
 ?>
 
-<!--<?php
-add_filter( 'wp_title', 'wpdocs_hack_wp_title_for_home' );
-
-/**
- * Customize the title for the home page, if one is not set.
- *
- * @param string $title The original title.
- * @return string The title to use.
- */
-function wpdocs_hack_wp_title_for_home( $title )
-{
-  if ( empty( $title ) && ( is_home() || is_front_page() ) ) {
-    $title = __( 'Home', 'textdomain' ) . ' | ' . get_bloginfo( 'description' );
-  }
-  return $title;
-}
+<!-- adding and sizing - posts with thumbnails -->
+
+<?php
+  add_theme_support( 'post-thumbnails' );
+  set_post_thumbnail_size( 100, 50, true );
```

Comment:
Don't add functions straight into functions.php. You want to have control over when these functions will be executed: 

``` 
add_action('init', 'setupshop_theme_init');
function setupshop_theme_init() {
  add_theme_support( 'post-thumbnails' ); 
  set_post_thumbnail_size( 100, 50, true );
}